model/span.go

```diff
@@ -0,0 +1,49 @@
+package model
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// SpanContext holds the context of a Span.
+type SpanContext struct {
+	TraceID  TraceID `json:"traceId"`
+	ID       ID      `json:"id"`
+	ParentID *ID     `json:"parentId,omitempty"`
+	Debug    bool    `json:"debug,omitempty"`
+	Sampled  *bool   `json:"-"`
```

Comment:
what does hyphen mean?

Reply:
It means even though the Sampled property is exported so other Go packages can directly set/read the property, the property is not to be JSON serialized / deserialized. If the hyphen would be omitted it would allow to bleed through in a JSON object or be set by a JSON deserialized object.